data/test_evaluations/form-end-to-end-stories.md

```diff
@@ -0,0 +1,37 @@
+## Happy path
+* greet: /greet
+    - utter_greet
+* request_restaurant: /request_restaurant
+    - restaurant_form
+    - form{"name": "restaurant_form"}
+    - form{"name": null}
+    - utter_slots_values
+* thankyou: /thankyou
+    - utter_noworries
+
+## Happy path with form prefix
+* greet: /greet
+    - utter_greet
+* request_restaurant: /request_restaurant
+    - restaurant_form
+    - form{"name": "restaurant_form"}
+* form: /inform{"cuisine": "afghan"}
```

Comment:
does it actually work? shouldn't the label before `:` be intent for correct intent classification?

Reply:
thanks for spotting this, it actually has to be `form: inform: /inform...`

Reply:
oh no. does it actually work this way?

Reply:
Yes, it works with `form: /inform ...` and `form: inform: /inform ...` (this was copied from the example we used before). Not sure about the notation.

Reply:
hm, why it works with `form: /inform ...` does it mean, it doesn't actually test intent prediction?

Reply:
Not sure how `form: inform:` gets parsed, but I assume the `inform:` is just ignored. The intent needs to be correctly recognized  otherwise the tests fails.